pulp_container/app/tasks/tag.py

```diff
@@ -27,10 +26,8 @@ def tag_image(manifest_pk, tag, repository_pk):
     if created:
         resource = CreatedResource(content_object=manifest_tag)
         resource.save()
-
-    ContentArtifact.objects.get_or_create(
-        artifact=artifact, content=manifest_tag, relative_path=tag
-    )
+    else:
+        manifest_tag.touch()
```

Comment:
good find!
Should we link this to #9134?

Reply:
i did reference 9134 so the revision is created however i think ci will still fail because of lack of changelong entry, i suggest to merge as admin

Reply:
Well, the changelog is there; just not in the same commit. XD

Reply:
hm ok, seems like ci logic is smart enough to handle this case, merging :)

Reply:
I'd say, not smart enough to _see_ it.